### DISCUSSION TITLE
useContractReads  how to custom the multicall address

### DISCUSSION THREAD

**Discussion by shifenhutu** [wevm/wagmi#2507] [Ideas]
the default multicall address is `0xca11bde05977b3631167028862be2a173976ca11`

I worry it's not safe

i want to change it in to my own address

is there any way to change it ?

**Comment by tmm** [comment]
You can [update the multicall contract address](https://wagmi.sh/react/chains#build-your-own) on whatever chain you are trying to use. For what it's worth, we think https://github.com/mds1/multicall is extremely credible (we wouldn't make it the default if we thought otherwise).
